roles/ovirt-engine-install-packages/defaults/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+# default vars for ovirt-ovirt-engine-install-packages-packages role
+ovirt_engine_dwh: True
+ovirt_engine_version: 4.1
+ovirt-engine-extension-aaa-ldap: True
```

Comment:
What about aaa-misc, logger etc? I just wonder whether it wouldn't be better to set a list of additional packages

Reply:
changing to list.